### PR TITLE
Add Image View & BlobMedia Hook

### PR DESCRIPTION
**Pull request by flyntdubu**

### CODE REVIEW

File: components/BlobMedia.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { encodeBase64 } from "../lib/storage/encoding";
 // @ts-ignore
 import { Image } from "native-base";
 
-export default function BlobMedia({ handle }: { handle: Handle }) {
+export function useMedia(handle: Handle): string | null {
```
Comment (wgoodall01) [inline components/BlobMedia.tsx:11]: This hook should probably be pulled out into its own file, in `./hooks`, for consistency. It's not strictly part of the `BlobMedia` component here

File: screens/ViewImageScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <Box>
+        <Spinner size="lg" />
+      </Box>
+    );
+  }
+
+  return (
+    <Center>
+      <Image
+        style={{ width: "100%", height: 500 }}
```
Comment (wgoodall01) [inline screens/ViewImageScreen.tsx:34]: Is there any way to make the image take up the entire screen vertically?

File: screens/ViewProfileScreen.tsx [hunk 3]
```diff
@@ -115,16 +117,24 @@ export default function ViewProfileScreen({
...
-              rounded={20}
+            <Pressable
+              onPress={() =>
+                navigation.navigate("ViewImage", { attachment: attachment })
+              }
             >
-              <BlobMedia handle={attachment.blob} />
-              <HStack justifyContent="center" alignItems="center"></HStack>
-            </Entry>
+              <VStack>
```
Comment (wgoodall01) [inline screens/ViewProfileScreen.tsx:125]: Why is there a `VStack` here with only one child?

File: screens/ViewProfileScreen.tsx [hunk 4]
```diff
@@ -115,16 +117,24 @@ export default function ViewProfileScreen({
...
+                <Entry
+                  space={2}
+                  px={2}
+                  py={2}
+                  key={attachment.blob.hash}
+                  rounded={20}
+                >
+                  <BlobMedia handle={attachment.blob} />
+                  <HStack justifyContent="center" alignItems="center"></HStack>
```
Comment (wgoodall01) [inline screens/ViewProfileScreen.tsx:134]: Why is there an empty `HStack` here?

File: package.json [hunk 5]
```diff
@@ -25,6 +25,7 @@
     "@react-navigation/native": "^6.0.2",
     "@react-navigation/native-stack": "^6.1.0",
     "@react-navigation/stack": "^6.0.11",
+    "accordion-collapse-react-native": "^1.1.0",
```
Comment (wgoodall01) [inline package.json:28]: You don't seem to be actually using this dependency. Please remove it and update the `yarn.lock`.

File: screens/ViewProfileScreen.tsx (hunk 4 above)
Comment (flyntdubu) [reply screens/ViewProfileScreen.tsx:134]: forgor

File: package.json (hunk 5 above)
Comment (flyntdubu) [reply package.json:28]: ok

File: screens/ViewImageScreen.tsx (hunk 2 above)
Comment (flyntdubu) [reply screens/ViewImageScreen.tsx:34]: I couldn't find anything in the docs. If you can feel free to change it though..!

File: screens/ViewProfileScreen.tsx (hunk 3 above)
Comment (flyntdubu) [reply screens/ViewProfileScreen.tsx]: There is no longer a `VStack` here with only one child.

**Review by wgoodall01 — APPROVED** [review]
Looks good!

**Pull request merged**